subjects/cybersecurity/obfuscator/audit/README.md

```diff
@@ -13,13 +11,12 @@ Knowledge-Check:
 Files that must be inside your repository:
 
 - Your program source code.
+
 - a README.md file, Which clearly explains how to use the program.
 
 ###### Does the required files present?
 
-##### Evaluate the student's submission
-
-Launch the student's program on the OS he wants
+##### Launch the student's program on the OS he wants
```

Comment:
The gender should be neutral, I think the best would be to repeat `the student` instead of `he`

Reply:
Or maybe `the learner` could be used as well.

Reply:
done